tests/unit/s2n_client_cert_verify_test.c

```diff
@@ -81,5 +146,51 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_cert_chain_and_key_free(chain_and_key));
     }
 
+    /*  Test: client certificate verify. */
```

Comment:
Now that this is in `s2n_client_cert_verify_test` you'll probably want to name this test case differently ;)

Reply:
Good point, renamed!